com.unity.formats.alembic/Editor/Importer/AlembicImporterEditor.cs

```diff
@@ -397,6 +407,141 @@ static Material LoadMaterial(string assetName, string[] searchPath)
 
             return AssetDatabase.LoadAssetAtPath<Material>(path);
         }
+
+        void DrawHairUI(AlembicImporter importer, bool isMultiEdit)
+        {
+            var hairLabel = L10n.Tr("Hair");
+            // var hairLabel = EditorGUIUtility.TrTextContentWithIcon("Hair",
+            //     EditorGUIUtility.IconContent("info").image); // Title with Icon
+
+            EditorGUILayout.LabelField(hairLabel, EditorStyles.boldLabel);
+            {
+                GUILayout.FlexibleSpace();
+
+#if !HAIR_AVAILABLE
+                string msg = L10n.Tr("There is no Hair package installed. " +
+                                     "Install the hair package first to generate " +
+                                     "a curve-based groom.");
+                                    // + "<a href=\"https://unity.com/\">Learn More</a>");
+
+                GUILayout.FlexibleSpace();
+                ButtonHelpbox(msg, L10n.Tr("Install"),
```

Comment:
Wording here might need revision since the button opens the github repo and doesn't actually install anything

Reply:
When there is no Hair package installed
![image](https://github.com/Unity-Technologies/com.unity.formats.alembic/assets/35614138/d83f369b-ba72-41c8-87f7-7deefafb09bd)